### ISSUE TITLE
Doesn't work with Safari?

### ISSUE THREAD

**Issue by grosch** [Wikiki/bulma-extensions#69]
If I use this in Chrome on an input it seems to work well.  If I open it in Safari it's just a text field.

I should not I have not figured out how to include the javascript yet.  I'm using angular, so in my angular.json I just add the bulmaCalendar.min.js file to the scripts area.  Not sure how to "call" it from my .ts file yet though.

**Comment by Wikiki** [comment]
Hi,

Please use the last release with the new <code>.attach()</code> method.

**Comment by grosch** [comment]
Can you help me with the syntax for that when I'm using this in an Angular application please?

**Comment by Wikiki** [comment]
See https://wikiki.github.io/components/calendar/

**Comment by grosch** [comment]
That's not a typescript example.  There's not a "bulma" object available for example, and if I call "new BulmaCalendar" still there's not a new `attach` method available.

**Comment by Wikiki** [comment]
As described in the documentation : don't call new but directly the .attach() method. It's pure javascript so you should be able to use it within your typescript code

**Comment by grosch** [comment]
If I just do `bulma.attach` then I get a typescript error on `bulma` not being defined.  I had bought your book on Bulma hoping for a bit more info in the Angular section as I'm likely doing things wrong. 

In my angular.json I added your stuff to styles and scripts:

```json
            "styles": [
              "node_modules/bulma/css/bulma.css",
              "node_modules/bulma-calendar/dist/css/bulma-calendar.min.css",
              "src/styles.css"
            ],
            "scripts": [
              "node_modules/bulma-calendar/dist/js/bulma-calendar.min.js"
            ]
```

Am I supposed to be doing something different to make that available in my component?

**Comment by Wikiki** [comment]
I think you're mixing Jeremy Thomas (Bulma's creator) and I (Bulma Extensions's creator).
You try to use bulma.attach() but the extension is called bulmaCalendar as described into the documentation. Please read the documentation carefully

**Comment by grosch** [comment]
Yes I was, sorry.  bulmaCalendar has the same issue from typescript of not being defined.

**Comment by Wikiki** [comment]
Can you please try to replace the source code int the file <code>node_modules/bulma-calendar/dist/js/bulma-calendar.min.js</code> with the source xode of the same file from this repositroy ? (To be sure it's not an issue with the npm version)

**Comment by grosch** [comment]
They're the same.  I think this just has something to do with the way the javascript is imported into an angular application.
